src/CoreWCF.BuildTools/src/SourceGenerationSpec.cs

```diff
@@ -14,15 +14,23 @@ namespace CoreWCF.BuildTools
         INamedTypeSymbol? TaskSymbol,
         INamedTypeSymbol? GenericTaskSymbol,
         INamedTypeSymbol? CoreWCFInjectedSymbol,
-        INamedTypeSymbol? MicrosoftAspNetCoreMvcFromServicesSymbol)
+        INamedTypeSymbol? MicrosoftAspNetCoreMvcFromServicesSymbol,
+        INamedTypeSymbol? RemoteEndpointMessagePropertySymbol,
+        INamedTypeSymbol? KafkaMessagePropertySymbol,
```

Comment:
We shouldn't be special casing the individual property types. There are many more property types than listed here and we shouldn't be limited to these types. This should be able to be implemented in a general way.